packages/db-kit/src/cli/decodeTransaction/useDecodedTransaction.ts

```diff
@@ -47,7 +47,7 @@ export function useDecodedTransaction({
   }
 
   useEffect(() => {
-    decoder.decodeTransaction(transaction).then(decoding => {
+    decoder.decodeTransaction(transaction as unknown as DecoderTransaction).then(decoding => {
```

Comment:
Is casting to unknown required here?

Reply:
Why @gnidan? What would happen if the interim cast to unknown?

Reply:
yeah, sinch `transaction` and `DecoderTransaction` were different and it couldn't be cast directly, we need to first type it as `unknow` then to `DecoderTransaction`

Reply:
I may need to update the `DecoderTransaction` type a bit with some of the new fields, dunno if that would help.  (Or @lsqproduction I don't know if you want to include that on this PR... looks to me like new fields are `type: number`, `maxFeePerGas: string`, and `maxPriorityFeePerGas: string`?  Dunno if that will actually improve things, as mentioned.)

Reply:
What's the typescript error without the interim cast? I removed the interim cast locally, and it compiled without errors. I'm not sure what I'm doing wrong. OR is this a case where we're concerned with converting primitive types that we need the interim cast to unknown?

Reply:
@lsqproduction and I discovered the truffle repo had an ancestor `$HOME/node_modules`, which breaks npm/yarn dependency guarantees. After removing the ancestor `node_modules`, we observed `yarn bootstrap` is consistent with other dev-machines. 

If you're curious you can try installing an older version of `web3` in an ancestor folder (`$HOME` should work!). 
```console
cd $HOME
npm install web3@1.5.0
cd path/to/your/truffle/repo
yarn bootstrap # should fail in db-kit
```

Reply:
Just to be clear, this really is necessary?